CONTRIBUTING.md

```diff
@@ -30,7 +30,7 @@ sudo apt-get install php7.0 php7.0-xml php7.0-mbstring php7.0-pgsql php7.0-gd ph
 ### Preparing Ubuntu 14.04 
 
 ```sh
-sudo apt-get install php5 php5-xml php5-mbstring php5-pgsql php5-gd php5-curl php5-cli php5-sqlite php5.0-intl sqlite curl openssl
+sudo apt-get install php5 php5-xml php5-mbstring php5-pgsql php5-gd php5-curl php5-cli php5-sqlite php5-intl sqlite curl openssl
 ```
```

Comment:
This will not work in a vanilla ubuntu 14.04 . For php5-intl you have to activate universe repo and php5-xml and php5-mbstrig are no installable packages.